### DISCUSSION TITLE
[Lesson8] anvil don't stop & Metamask keep loading issue

### DISCUSSION THREAD

**Discussion by LareineHan** [Cyfrin/foundry-full-course-cu#872] [Q&A] [answered]
Hello, so I have issue when I run 
```zsh
make anvil
```
on 
```zsh
foundry-fund-me-f23
```
folder. 

It just don't stop loading like the picture below. 
<img width="334" alt="Screenshot 2023-09-14 at 12 56 48 PM" src="https://github.com/Cyfrin/foundry-full-course-f23/assets/128317012/88b64ea8-5a49-49c8-8fb5-7a918634298d">
<img width="499" alt="Screenshot 2023-09-14 at 12 56 36 PM" src="https://github.com/Cyfrin/foundry-full-course-f23/assets/128317012/0699449b-9b74-4fb5-8941-747b3483a1a0">

Additionally, when I add anvil's private key to my MetaMask as import account, it shows loading icon and don't really work nor adding any fund init. 
<img width="342" alt="Screenshot 2023-09-14 at 12 56 14 PM" src="https://github.com/Cyfrin/foundry-full-course-f23/assets/128317012/711f8867-7479-4a6c-8bce-119e3f0ae5af">

Can anyone identify the problem please? 
Thank you community! :D

**Comment by 0xusmanf** [comment, marked as answer]
Try running anvil with only `anvil` command instead of `make anvil`.

And I guess you have to add anvil as a network to your meta mask and then you can import an anvil account.

Edit: I see you have already added anvil as a network in your metamask, run anvil first by `anvil` command and then import the account.
